README.md

```diff
@@ -54,6 +54,14 @@ The `dev` mode creates an application build that has been optimized for debuggin
 
 The `test` mode creates bundles that can be used to run the unit and functional tests of the application.
 
+### Watching
+
+The mutually-exclusive `--watch` (or `-w`) and `--serve` (or `-s`) flags are provided to automatically recompile after the source changes. The `--watch` flag is available in every mode (see above), while the `--serve` flag forces `dev` mode. `--watch` simply recompiles in response to source changes and writes directly to the disk. The `--serve` flag, on the other hand, starts a webserver and handles all files in memory. By default, the application is served on port 9999, but this can be changed with the `--port` (`-p`) flag:
```

Comment:
seems like a `--serve` mode would potentially be useful for a file based watch too? like a basic static server?

Reply:
Do you have a different flag in mind?

Reply:
After discussing this directly with @matt-gadd and @agubler, we've settled on using `--serve` just to start a server regardless of how file watching is handled. `--watch` will be expanded to `--watch=file|memory` to control whether the `webpack-dev-middleware` (`--watch=memory`) or `compiler.watch` (`--watch=file`) is used.